app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -1214,6 +1214,160 @@ class PinPasswordActivityTest {
     }
   }
 
+  @Test
+  fun testPinPassword_withAdmin_inputPin_configChange_inputPinIsPersisted() {
+    ActivityScenario.launch<PinPasswordActivity>(
+      PinPasswordActivity.createPinPasswordActivityIntent(
+        context = context,
+        adminPin = adminPin,
+        profileId = adminId
+      )
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(withId(R.id.pin_password_input_pin_edit_text))
+        .perform(editTextInputAction.appendText("1234"), closeSoftKeyboard())
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.pin_password_input_pin_edit_text))
+        .check(matches(withText("1234")))
```

Comment:
Admin PIN is always 5 digits. It is ok, but not necessary, to test for 4 digits because the next test already checks for 3 digits. The tests we are interested in are:
1. Empty input(I think it is already covered in existing tests)
2. inputs under 3 digits
3. Inputs over 5 digits
4. 3 digits 
5. 5 digits